scripts/rnnlm/get_word_features.py

```diff
@@ -41,7 +41,7 @@
 # return the vocab, which is a dict mapping the word to a integer id.
 def read_vocab(vocab_file):
     vocab = {}
-    with open(vocab_file, 'r', encoding="latin-1") as f:
+    with open(vocab_file, 'r', encoding="utf-8") as f:
         for line in f:
             fields = re.split(tab_or_space, line)
```

Comment:
and this tab_or_space stuff would disappear if using utf-8 encoding, it would just be line.split()